corehq/apps/cleanup/tests/util.py

```diff
@@ -47,3 +50,8 @@ def get_cleaned_couch_attrs(cls, couch_model_cls):
 
 def is_monday():
     return datetime.utcnow().isoweekday() == 1
+
+
+@unit_testing_only
+def delete_all_deleted_sql_docs():
+    DeletedSQLDoc.objects.all().delete()
```

Comment:
This utility should not be necessary, assuming `DeletedSQLDoc` is in pgmain. SQL tests use transactions, which are rolled back to automatically clean up any objects created by the test.